core/adapters/bridge_test.go

```diff
@@ -61,7 +61,6 @@ func TestBridge_IncludesMetaIfJobRunIsInDB(t *testing.T) {
 	blockHash, _ := hex.DecodeString(blockHashHex)
 	store, jobRunID, cleanup := setupJobRunAndStore(t, txHash, blockHash)
 	defer cleanup()
-	store.Config.Set("BRIDGE_RESPONSE_URL", cltest.WebURL(t, ""))
```

Comment:
Are the changes in this file indicated by the race detector?

Reply:
Yes, setting config is racy if done at the wrong time.

Reply:
Because it is called after app.Start(). And the application reads this config value on boot. So the read races with the write, unless we move the `Set` to before the app is started.